src/jobs/invoices.js

```diff
@@ -0,0 +1,52 @@
+const StripeLib = require('../libs/stripe-lib');
+const { TransactionModel, OrderModel, ProductModel } = require('../models/index');
```

Comment:
no need to add index when you import from dir index file, it automatically looks for it first